app/main.py

```diff
@@ -1,6 +1,6 @@
 class Distance:
 
-    def __init__(self, km):
+    def __init__(self, km: float):
```

Comment:
And also add type annotation for constructor and all methods. What they return. Like so __init(.....) -> None:
__add__(...) -> float:

Reply:
agree